src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
```

Comment:
> Later, I would like to [...]

That will be later. Please remove one of them for that PR :-)

> [...] generate the DSA key oportunisticly (it adds delay on shout startup currently)

Ouch.
Opportunistically or not, this should be an asynchronous call IMO (for a future PR, synchronous here is fine). A promise, perhaps... :-)